hack/tests/check-lint.sh

```diff
@@ -53,4 +53,5 @@ golangci-lint run --disable-all \
     --enable=dupl \
     --enable=unparam \
     --enable=golint \
+    --exclude [a-z] \
```

Comment:
what does it mean (`exclude [a-z]`)?
Could you please also add a comment?
Also, shows that is missing add the lint which is the motivation of this PR. Could you please check?

Reply:
it excludes  cretain lines from --lll

Reply:
But I understand that it will not just be used/valid for the specific check. 
I think the best way is we ignore files and dirs in the config. WDYT?

Reply:
there are files that need to be skipped for -lll, since they affect other tests if changed

Reply:
I understand. But if we are not able to exclude these files just for the check lll then I think would be more valuable we do not add/use it at all.  

By checking the doc [here](https://github.com/golangci/golangci-lint#config-file) shows that with the config file would be possible to exclude files, directories just for one specific lint. 

If you see that it will not work, then I think that the best approach would be we not add this check. WDYT? Is it make sense?

Reply:
You removed the exclude .. So closing this one as solved.